task/jib-maven/0.3/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+TMD=$(mktemp -d)
+
+# Generate SSL Certificate
+openssl req -newkey rsa:4096 -nodes -sha256 -keyout "${TMD}"/ca.key -x509 -days 365 \
```

Comment:
we should probably make that as default for add_sidecar_registry or add it as a add_sidecar_secure_registry if that's problematics for other tasks...

Reply:
Sure, I can add a new func for secure registry and use that in all tasks, may be will handle it in different PR with changes in all taks

Reply:
I have done the changes in the same PR now